parm/config/gfs/config.resources

```diff
@@ -595,8 +595,8 @@ case ${step} in
     ntasks=16
     case ${OCNRES} in
       "025")
-        ntasks=480
-        memory="256GB"
+        ntasks=800
+        memory="500GB"
```

Comment:
IIRC, marineanlletkf was descoped from GFSv17.

Reply:
This was changed prior to descoping.  This change can be reverted.